clang/lib/AST/ByteCode/Interp.cpp

```diff
@@ -1430,6 +1434,10 @@ bool Interpret(InterpState &S, APValue &Result) {
     }
   }
 }
+// https://github.com/llvm/llvm-project/issues/102513
+#if defined(_MSC_VER)
```

Comment:
Please make the ifdef conditions match, or define your own macro like DEOPTIMIZE_SWITCH. As written, I'd expect to see some clang-cl build warnings.